final/MealsharingNET/Controllers/MealController.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.AspNetCore.Mvc;
+using MealsharingNET.Models;
+namespace MealsharingNET.Controllers;
+
+[ApiController]
+[Route("meals")]
+public class MealController : ControllerBase{
```

Comment:
Try to use the auto formatting functionality to make the code more readable :) It's Alt + Shift+ F I think